lib/actions.js

```diff
@@ -42,6 +42,10 @@ class Actions extends Resources {
 
     if (options.action instanceof Buffer) {
       body.exec.code = options.action.toString('base64')
+      if (body.exec.kind == "java") {
+        body.exec.jar = body.exec.code;
```

Comment:
for reference https://github.com/openwhisk/openwhisk/blob/master/common/scala/src/main/scala/whisk/core/entity/Exec.scala#L241.